src/secp256k1.c

```diff
@@ -254,7 +254,10 @@ static int nonce_function_rfc6979(unsigned char *nonce32, const unsigned char *m
    return 1;
 }
 
+SECP256K1_API
 const secp256k1_nonce_function_t secp256k1_nonce_function_rfc6979 = nonce_function_rfc6979;
+
+SECP256K1_API
 const secp256k1_nonce_function_t secp256k1_nonce_function_default = nonce_function_rfc6979;
```

Comment:
I have not had a chance to verify these two lines, but according to documentation it appears correct for both VC and GCC.